aws_signing_helper/signer.go

```diff
@@ -747,5 +741,18 @@ func ReadCertificateData(certificateId string) (CertificateData, error) {
 	}
 
 	//return struct
-	return CertificateData{keyType, encodedDer, serialNumber, supportedAlgorithms}, nil
+	return CertificateData{keyType, encodedDer, serialNumber, supportedAlgorithms}, cert, nil
+}
+
+// GetCertChain Read a certificate bundle and return a chain of all the certificates is contains
```

Comment:
Minor, but there's a typo in the comment here (the `is contains` at the end).

Reply:
Fixed